equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -167,6 +172,21 @@ public <T extends CrmFormModelInterface> T convertCrmFileJsonToModel(JSONObject
         return (T) crmFormData;
     }
 
+
+    public void sectionsCheck(JSONObject crmFileJsonObject) {
+        JSONArray sectionsArray  = (JSONArray)((JSONObject)(crmFileJsonObject.getJSONObject(FORM_SECTION_STATES))).get(FORM_SECTION_NAMES);
```

Comment:
I don't think this is a logic that should be in here. And it is messy.  It looks like we have 2 logics competing with each other to determine if it is a CRM15. 
Look at ticket EMP-604. And let's wait for its solition

Reply:
I just saw this new ticket. Lets hold untill we decide on EMP-604